src/panels/AuditLogsFilterPanel/Fields/Fields.tsx

```diff
@@ -0,0 +1,117 @@
+import { Input } from "@canonical/react-components";
+import { Field, useFormikContext } from "formik";
+import { useSelector } from "react-redux";
+
+import AutocompleteInput from "components/AutocompleteInput";
+import {
+  getAuditEventsFacades,
+  getAuditEventsMethods,
+  getAuditEventsUsers,
+  getAuditEventsModels,
+  getModelNames,
+  getUsers,
+} from "store/juju/selectors";
+
+import type { FormFields } from "../types";
+
+export enum Label {
+  AFTER = "After",
+  BEFORE = "Before",
+  FACADE = "Facade",
+  METHOD = "Method",
+  MODEL = "Model",
+  USER = "User",
+  VERSION = "Version",
+}
+
+const Fields = (): JSX.Element => {
```

Comment:
These fields are in a separate component from the form so that we can use `useFormikContext`.